bigquery_datatransfer/samples/tests/test_create_scheduled_query.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import os
+
+import google.api_core.exceptions
+import google.auth
+import google.cloud.bigquery
+import pytest
+
+from .. import create_scheduled_query
+
+
+@pytest.fixture
+def project_id():
+    return os.environ["PROJECT_ID"]
+
+
+@pytest.fixture(scope="module")
+def credentials():
+    creds, _ = google.auth.default(["https://www.googleapis.com/auth/cloud-platform"])
```

Comment:
Should we add some breadcrumbs about the permission setup for making the service account work here?

Reply:
The error messages actually showed me the commands to run this time, so that was good. Will the instructions change when it starts working how it's supposed to work?

Reply:
Done.